woocommerce/woocommerce-gateway-stripe
# fix: dupe payment request height

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1923

Removes the duplicated height based on the UPE feature flag.

# Testing instructions
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- The height/size attribute on the payment request section should no longer appear duplicated
